lukasz-lapczyk/review-dealer
# Add review editing feature

# Overview
Users can edit their reviews. Each review is being tracked for changes. Version history of each review is available for all users at the review page, while the most common is being displayed on company page (marked as "(edited)" if any changes were made). 

# Testing prerequisites
- bundle install
- db:migrate
- db:seed

# Testing
- navigate to root directory
- log in as John Doe with email: 'user_1@example.org', password: Secret19
- click on 'Foobar #2' at the company listing
- click on 'Show' button next to your (John Doe's) review
- click on 'Edit' button below your review ID
- update review by filling in the form with valid data
- click 'Update' button
- [x] updated review should appear on listing, marked as "(edited)"
- click on 'Show' button next to your review
- click on "Ediding history" link below your review
- [x] previous version of review should appear
